src/Microsoft.PowerApps.TestEngine/PowerApps/PowerAppFunctions.cs

```diff
@@ -190,10 +190,8 @@ public async Task<bool> SetPropertyAsync(ItemPath itemPath, FormulaValue value)
 
             ValidateItemPath(itemPath, false);
             // TODO: handle components
-            var itemPathString = JsonConvert.SerializeObject(itemPath);
-            var argument = new string[] { itemPathString, objectValue.ToString() };
-            var expression = "([itemPathString, objectValue]) => PowerAppsTestEngine.setPropertyValue(itemPathString, objectValue)";
-            return await _testInfraFunctions.RunJavascriptAsync<bool>(expression, argument);
+            var expression = $"PowerAppsTestEngine.setPropertyValue({JsonConvert.SerializeObject(itemPath)}, {JsonConvert.SerializeObject(objectValue)})";
+            return await _testInfraFunctions.RunJavascriptAsync<bool>(expression);
```

Comment:
I think we might also need to do this for SetPropertyDateAsync, SetPropertyRecordAsync, and SetPropertyTableAsync. I'm a bit worried that we're constructing the JSON object partially via string concatenation rather than using the JsonConvert to ensure we don't have any injection vulnerabilities that we haven't caught yet (or may expose in the future)